mithril-aggregator/src/http_server/routes/root_routes.rs

```diff
@@ -182,11 +192,18 @@ mod tests {
     async fn test_root_route_ok_with_cardano_transactions_enabled() {
         let method = Method::GET.as_str();
         let path = "/";
-        let mut dependency_manager = initialize_dependencies().await;
-        dependency_manager
-            .signed_entity_config
-            .allowed_discriminants =
-            BTreeSet::from([SignedEntityTypeDiscriminants::CardanoTransactions]);
+        let config = Configuration {
+            signed_entity_types: Some(
+                BTreeSet::from([SignedEntityTypeDiscriminants::CardanoTransactions])
+                    .iter()
+                    .map(SignedEntityTypeDiscriminants::to_string)
+                    .collect::<Vec<_>>()
+                    .join(","),
+            ),
```

Comment:
Is there any purpose of using this construction instead of a simpler like the following ?
```suggestion
            signed_entity_types: Some(format!("{}", SignedEntityTypeDiscriminants::CardanoTransactions)),
```

Reply:
Done ✅